onnx/defs/operator_sets.h

```diff
@@ -727,6 +728,7 @@ class OpSet_Onnx_ver12 {
     fn(GetOpSchema<ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 12, MaxPool)>());
     fn(GetOpSchema<ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 12, ReduceMax)>());
     fn(GetOpSchema<ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 12, ReduceMin)>());
+    fn(GetOpSchema<ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 12, Celu)>());
```

Comment:
Please split this line following his friends above.

Reply:
Could you develop? I don't understand which friends. 🙂

Reply:
On Github the lines appear split but in the actual file (from current master), each op is on a sigle line. Or did I missed something?